pkg/sql/opt/exec/explain/emit.go

```diff
@@ -31,6 +32,14 @@ func Emit(plan *Plan, ob *OutputBuilder, spanFormatFn SpanFormatFn) error {
 	e := makeEmitter(ob, spanFormatFn)
 	var walk func(n *Node) error
 	walk = func(n *Node) error {
+		// In non-verbose mode, we skip all projections.
+		// In verbose mode, we only skip trivial projections (which just rearrange
+		// or rename the columns).
+		if !ob.flags.Verbose {
+			for n.op == serializingProjectOp || n.op == simpleProjectOp {
+				n = n.children[0]
```

Comment:
just a nit, but I think that `return walk(n.children[0])` is cleaner.

also, this is a pretty hacky way to get recursion :D though it is the common example of how a programming language with only functions and mutable memory can get you turing completeness